index.js

```diff
@@ -278,6 +283,18 @@ function alive (list) {
   return true
 }
 
+function isDelayInvalid (ms) {
+  if (ms < 1 || ms > 2147483647) return true
```

Comment:
setTimeout(0) is legal afaik

Reply:
It is, but Node will set a delay of `1` when the value is zero or negative